Classes/Command/MigrationCommandController.php

```diff
@@ -164,8 +164,8 @@ protected function migrateSqlFile(SplFileInfo $fileinfo, &$errors, &$output)
 
         // check TYPO3 version
         $currentVersion = VersionNumberUtility::convertVersionStringToArray(VersionNumberUtility::getCurrentTypo3Version());
-        if (isset($currentVersion['version_main']) && $currentVersion['version_main'] === 8) {
-            /* new 8.7 */
+        if (isset($currentVersion['version_main']) && $currentVersion['version_main'] >= 9) {
```

Comment:
Why `>= 9` and not `>= 8` ?

Reply:
Because it should work in version 8 and above. (not only in 9 and above)